src/main/java/com/project/momo/common/annotation/validator/OauthTypeValidator.java

```diff
@@ -9,6 +9,7 @@
 
     @Override
     public boolean isValid(String value, ConstraintValidatorContext context) {
+
         for (OauthType oauthType : OauthType.values()) {
             if(oauthType.getValue().equals(value)) return true;
         }
```

Comment:
```suggestion
        return Arrays.stream(OauthType.values())
                .anyMatch(oauthType -> oauthType.getValue().equals(value));
```
으로도 쓸 수 있을 것 같네요.

그리고 해당 로직은 `OauthType.java`에서 관리하는게 올바른 책임이 아닐까요?

Reply:
네, String 타입으로 들어온 값이 `OauthType`에 포함되는지 검증하는 로직은 `OauthType.java`의 책임이 맞다고 생각합니다. 수정하겠습니다!!